packages/storybook/src/nimble/select/select-opened-matrix.stories.ts

```diff
@@ -7,6 +7,7 @@ import { FilterMode } from '../../../../nimble-components/src/select/types';
 import { createFixedThemeStory } from '../../utilities/storybook';
 import { sharedMatrixParameters } from '../../utilities/matrix';
 import { backgroundStates } from '../../utilities/states';
+import { isChromatic } from '../../utilities/isChromatic';
```

Comment:
Reminder to take some care when Chromatic diffs. It looks like [the current build](https://www.chromatic.com/test?appId=60e89457a987cf003efc0a5b&id=6669161f7a2ba05516df3199) is exhibiting #1862 where sub menus are positioned far away from their parent, so we don't want to set that as the new baseline. My best advice is just to trigger a new build.